x/oracle/genesis_test.go

```diff
@@ -0,0 +1,57 @@
+package oracle_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/medibloc/panacea-core/v2/types/testsuite"
+	"github.com/medibloc/panacea-core/v2/x/oracle"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+	"testing"
+)
+
+var oracle1 = sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address())
+
+type genesisTestSuite struct {
+	testsuite.TestSuite
+}
+
+func TestGenesisTestSuite(t *testing.T) {
+	suite.Run(t, new(genesisTestSuite))
+}
+
+func (suite genesisTestSuite) TestDataPoolInitGenesis() {
+	var oracles []types.Oracle
+
+	tempOracle := makeSampleOracle()
+
+	oracles = append(oracles, tempOracle)
+
+	genState := &types.GenesisState{
+		Oracles: oracles,
+	}
+
+	oracle.InitGenesis(suite.Ctx, suite.OracleKeeper, *genState)
+
+	// check oracle
+	oracleFromKeeper, err := suite.OracleKeeper.GetOracle(suite.Ctx, oracle1)
```

Comment:
Let's check by getting all the oracles from the keeper and its length to be equal to 1.